extensions/gdx-tools/src/com/badlogic/gdx/tools/hiero/Hiero.java

```diff
@@ -436,7 +436,14 @@ private void initializeEvents () {
 			public void valueChanged (ListSelectionEvent evt) {
 				if (evt.getValueIsAdjusting()) return;
 				prefs.put("system.font", (String)fontList.getSelectedValue());
-				updateFont();
+				try {
+					updateFont();
+				} catch (GdxRuntimeException ex) {
+					prefs.remove("system.font");
+					fontList.setSelectedValue("Arial", true);

```

Comment:
I would prefer to select the first available value instead. Did you try if that works?

Reply:
The first available value isn't guaranteed to have the 'x' either, depending on what fonts the user has installed. Either this could loop until it finds a valid font (possibly never, but that shouldn't ever happen...), or it could use a font we know, or at least believe, is there already, and is valid.  Hiero already falls back to Arial in one case.

Reply:
You are right. What happens if you select such a font while using Hiero? It will also switch back to Arial. I am not sure if this is a good UX, maybe it is enough to just show the error to the user.

Reply:
I don't think it should permit such fonts to stay selected; they break at least the Freetype font generator, maybe also Java and Native. It wouldn't be able to show a sane glyph window, though the sample text could show a bunch of not-supported squares (but it wouldn't have a readable error message, which is another reason why it needs to switch to a font that supports at least ASCII).